CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ MESSAGE(STATUS "====================================================")
 # Set the name of the project and target:
 SET(TARGET "aspect")
 
-FILE(GLOB_RECURSE TARGET_SRC  "source/*.cc" "unit_tests/*.cc" "include/*.h" "contrib/catch/catch.hpp")
+FILE(GLOB_RECURSE target_src  "source/*.cc" "unit_tests/*.cc" "include/*.h" "contrib/catch/catch.hpp")
 INCLUDE_DIRECTORIES(${CMAKE_BINARY_DIR}/include include  contrib/catch)
```

Comment:
I am used for cmake stuff to make it capital, so it is consistent with with cmake does in for example this line and also the deal.ii cmake variables like _DEALII_GOOD in line 81, but looking at for example line 95, we are not really consistent anyway.

I am not sure what the best approach is here, but currently capital variables for cmake has my preference.

Reply:
cmake is case insensitive, so it doesn't matter. It's really just a convention that all commands and keywords are upper case and all of our own variables are lower-case. I agree that we're probably still inconsistent -- we're just moving in the "more consistent" direction with this patch.

Reply:
> cmake is case insensitive

That is not correct: https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#variables

We use lower and upper case variables all over the place in deal.II and in ASPECT. I don't feel strongly about it, but I think this is too much churn if we want to change that everywhere.

The TARGET_SRC variable is capitalized the same way in the deal.II CMakeFiles. :-)

Reply:
I was imprecise: cmake command names are case insensitive. 

But `TARGET_SRC` is not a built-in cmake variable -- it's one exclusively used in user projects, and I find it useful to distinguish between cmake commands and built-in stuff and user stuff using the case in which things are written. We may not be entirely consistent about it, but we can at least strive for it and get closer to it :-)

Reply:
What an important change for pull request #3000 :-)

I still think that it is weird to not consistently change capitalization in deal.II as well. I don't think you want to do this in the deal.II cmake code-base?

Reply:
Happy to do it in deal.II as well -- give me a minute!

Reply:
See https://github.com/dealii/dealii/pull/8283

Reply:
My point is: Matthias uses uppercase variables all over the place inside deal.II as well. Good luck convincing him. I am not opposed to the change, but I also don't see great value in doing so.

Reply:
Fair enough. Let's put this on hold for the moment. I didn't realize it was something so much larger than just this one variable.